### PR TITLE
Add target to run rootfull test container

### PR DESCRIPTION
**Pull request by tupyy**
This PR adds a new target `test-docker` which executes all the test in a rootfull contiainer.

### CODE REVIEW

File: tools/Dockerfile_test [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM fedora:35
```
Comment (jordigilh) [inline tools/Dockerfile_test:1]: not f36?

Comment (tupyy) [reply tools/Dockerfile_test:1]: I wanted to have go1.16 but I can upgrade. No problem

File: internal/workload/manager_test.go [hunk 2]
```diff
@@ -142,6 +143,10 @@ var _ = Describe("Manager", func() {
 			err = os.Chmod(datadir, 0444)
 			Expect(err).NotTo(HaveOccurred())
 
+			if common.IsContainerized() {
+				datadir = "/sys" // set to /sys which is always readonly
```
Comment (masayag) [inline internal/workload/manager_test.go:147]: why creating a directory without write permissions will not work here in this case?

Comment (tupyy) [reply internal/workload/manager_test.go:147]: because the containers runs with `CAP_DAC_OVERRIDE`

**Review by masayag — APPROVED** [review]

**Pull request merged**